File: src/python/ch01/1.2/l0/exercises_starter.py
def exercise_1():
    return 326 + 589

def exercise_2():
    return 704 - 259

def exercise_3():
    return 38 * 12

def exercise_4():
    q, r = divmod(123, 7)
    return q, r

def main():
    print("Exercise 1 (326+589):", exercise_1())
    print("Exercise 2 (704-259):", exercise_2())
    print("Exercise 3 (38*12):", exercise_3())
    q, r = exercise_4()
    print("Exercise 4 (123//7, 123%7):", q, r)


if __name__ == "__main__":
    main()

File: src/go/ch01/1.2/l0/exercises_starter.go
package main

import "fmt"

func exercise1() int { return 326 + 589 }
func exercise2() int { return 704 - 259 }
func exercise3() int { return 38 * 12 }
func exercise4() (q, r int) {
	return 123 / 7, 123 % 7
}

func main() {
	// Run binary demo first (separate from basic ops main output)
	demoBinary()

	fmt.Println("Exercise 1 (326+589):", exercise1())
	fmt.Println("Exercise 2 (704-259):", exercise2())
	fmt.Println("Exercise 3 (38*12):", exercise3())
	q, r := exercise4()
	fmt.Println("Exercise 4 (123//7, 123%7):", q, r)
}
